src/plugins/orangehrmPerformancePlugin/config/routes.yaml

```
@@ -184,17 +184,17 @@ performance_review_edit:
   methods: [ GET ]
 
 performance_tracker_list:
-  path: /performance/addPerformanceTracker
+  path: /performance/searchPerformanceTracker
```

```suggestion
  path: /performance/viewPerformanceTracker
```
how it work with menu items?